projects/ngx-prx-styleguide/src/lib/tz-datepicker/tz-datepicker.component.ts

```diff
@@ -116,22 +119,20 @@ export class TzDatepickerComponent implements OnInit {
         publishReplay(1),
         refCount()
       )
-      
-      .pipe(share());
```

Comment:
No more sharing?

Reply:
This was actually the culprit of the double http request. Sharing the Observable caused it to completely reinitialize every time it was subscribed to. Now that it's set to replay itself once (and `refCount` is in charge of keeping it hot as long as there's a subscriber), this doesn't do anything useful.

Reply:
Ah, okay.  Observables almost make sense when you spell it out for me!